src/credentials/FabricTable.h

```diff
@@ -140,6 +140,7 @@ class DLL_EXPORT FabricInfo
         ReleaseOperationalCerts();
     }
 
+    NodeId GetNodeId() const { return mOperationalId.GetNodeId(); }
```

Comment:
This makes me a bit nervous because I'm worried that folks will use this by accident in place of the PeerId. Maybe I'm being paranoid, but would you mind moving the .GetNodeId() call to the place where this is used in the display screen?

Reply:
Fwiw, I agree with @turon here.  Asking "what is my node id on this fabric" is something we need in several places and people are perennially confused by the GetPeerId bit.

Reply:
Ok, I can add a todo to rename `PeerId` -> `OperationalId`.